docs/events.md

```diff
@@ -429,6 +431,148 @@ croct.track('cartViewed', {
 ```
 </details>
 
+### cartModified
+
+This event records the user modified the shopping cart.
```

Comment:
You should track this event when a user modifies the shopping cart page, either by adding, removing, or updating items.